### DISCUSSION TITLE
dde-launcher background

### DISCUSSION THREAD

**Discussion by bigbruno** [linuxdeepin/developer-center#2852] [Features Request & Ideas | 特性请求 & 头脑风暴]
I think add options for background ir a good idea.

Point 1, if background is too brightness, ir very hard to ready name of apps, because text white in background semi white.

Point 2, for use in old PCs, like 1GB of RAM, if use background image, dde-launcher use 71 MB, but if set just one black pixel of wallpaper in deepin, dde-launcher use only 21 MB.

I think default effect of dde-launcher with 30% or 50% less brighness make, like dde-dock, see in image, right side is very hard to read in various monitors, but dde-dock is less brighness, if dde launcher use similar effect make better to read.

And add one option to disable background and use just a black background, maybe just enabled in dconf-editor or gconf, for use in old PCs with very low RAM.

**Comment by bigbruno** [comment]
If scale this pixel for fullhd black image, dde-launcher use 35 MB

**Comment by SalahAdDin** [comment]
:+1:

**Comment by Lokke** [comment]
i was able to reproduce this bug.
If you define a background image that does not contain any colors but only grayscale, the dde-launcher background becomes white in fullscreen mode.
![16d07432351c1c7c9e3147f8b4fe512c](https://user-images.githubusercontent.com/6406275/33154050-f07f1874-cfe5-11e7-8a09-606f6be5a290.png)
![deepinbildschirmfoto_20171123003608](https://user-images.githubusercontent.com/6406275/33154107-5637fee2-cfe6-11e7-8ff6-8e6e31b1b977.png)

i'm running deepin-launcher 4.2.5-1 from AUR

**Comment by sbwtw** [comment]
@Lokke It happens because of the blured image need more times to generate. but the non-blured image should be using. @kirigayakazushin

**Comment by bigbruno** [comment]
Now that we have the effect of live transparency using blur, I think it would be advantageous to remove the use of blurred wallpaper and use real transparency, as it is used in the launcher when it is used in reduced mode or in the dde-control-center.

**Comment by justforlxz** [comment]
Sorry, this issue will be closed soon. If it is necessary to discuss it again, please create a new issue.

**Comment by SalahAdDin** [comment]
> Now that we have the effect of live transparency using blur, I think it would be advantageous to remove the use of blurred wallpaper and use real transparency, as it is used in the launcher when it is used in reduced mode or in the dde-control-center.

well, they didn't care about this suggestion.

**Comment by justforlxz** [comment]
> > Now that we have the effect of live transparency using blur, I think it would be advantageous to remove the use of blurred wallpaper and use real transparency, as it is used in the launcher when it is used in reduced mode or in the dde-control-center.
> 
> well, they didn't care about this suggestion.

I will implement it.
